lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -2244,25 +2250,27 @@ class SqlParser(private val ion: IonSystem) : Parser {
         return ParseNode(ARG_LIST, null, items, rem)
     }
 
+
     /**
-     * Checks that the given [Token] list does not have any top-level tokens outside of the top level query. Throws
-     * an error if so.
-     *
-     * Currently only checks for `EXEC`. DDL and DML along with corresponding error codes to be added in the future
-     * (https://github.com/partiql/partiql-lang-kotlin/issues/354).
+     * Checks if the [ParseNode] requires to be only at top level. DML keyword tokens, for example, requires to be top level.
```

Comment:
super nit: at least for me the kdoc wording is confusing. From the implementation, it checks the ParseNode's parse type is one of the top level types right? The first sentence could be something like:
```kotlin
/**
 * Returns true if and only if this [ParseType] is one of the top level types...
 */
```
For the second sentence, it might be helpful to also mention that DML can have `FROM` as the top-level node.

Reply:
I will update it. Thanks.